hawkular-alerts-actions-plugins/hawkular-alerts-actions-email/src/test/resources/template.html.default_en_US.ftl

```diff
@@ -147,15 +147,15 @@
                                                 <tr>
                                                     <td align="center" style="color:#333333; font-family:Open sans,sans-serif; font-size:13px; line-height:21px;">
                                                         Acknowledge time:
-                                                        <span style="font-size:15px;">${alert.ackTime?number_to_datetime}</span>
+                                                        <span style="font-size:15px;">${alert.currentLifecycle.stime?number_to_datetime}</span>
```

Comment:
No, it's on purpose. as we use the same alert currentLifecycle.user to get the user.
So I guess that maintaining same terminology in the template is better for clarity.